packages/hydrogen/edge-entry.js

```diff
@@ -0,0 +1,12 @@
+import handleRequest from '__RELATIVE__/src/App.server';
+import indexTemplate from '__RELATIVE__/dist/client/index.html?raw';
+
+// ReadableStream is bugged in Vercel Edge, overwrite with polyfill
+import { ReadableStream } from 'web-streams-polyfill/ponyfill';
```

Comment:
Do we need to add `web-streams-polyfill` as a dependency to `@vercel/hydrogen`?

Reply:
It's a [dependency of Hydrogen itself](https://github.com/Shopify/hydrogen/blob/v1.x-2022-07/packages/hydrogen/package.json#L145).

```
$ npm ls web-streams-polyfill
hydrogen-app@0.0.0 /Users/nrajlich/hydrogen-app
├─┬ @shopify/cli-hydrogen@3.0.25
│ ├─┬ @shopify/cli-kit@3.0.25
│ │ └─┬ node-fetch@3.2.6
│ │   └─┬ fetch-blob@3.1.5
│ │     └── web-streams-polyfill@3.2.1 deduped
│ └─┬ @shopify/hydrogen@0.26.1
│   └── web-streams-polyfill@3.2.1 deduped
└─┬ @shopify/hydrogen@1.0.2
  └── web-streams-polyfill@3.2.1
```